segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/rocksdb/RocksDBConfig.java

```diff
@@ -69,6 +70,13 @@
     @Getter
     private final boolean directReads;
 
+    /**
+     * Enabling memory only will let rocksdb write everything into memory instead of disks. This supposed to provide
```

Comment:
Minor: `rocksdb` -> `RocksDB`, `This (is) supposed`.

Reply:
Got it